File: Models/Dtos/CommentDto.cs
using Newtonsoft.Json;

namespace PicturesAPI.Models.Dtos;

public class CommentDto
{
    public string Id { get; set; }
    public string PictureId { get; set; }

    public string Text { get; set; }
    public DateTime CommentAdded { get; set; }

    public AccountPreviewDto AccountPreview { get; set; }
    public bool IsModifiable { get; set; } = false;
    public bool IsAdminModifiable { get; set; } = false;
}

File: PooPosting/PooPosting.View/src/app/shared/utils/dtos/CommentDto.ts
import {AccountPreviewDto} from "./AccountPreviewDto";

export interface CommentDto {
  id: string;
  pictureId: string,
  text: string,
  isModifiable: boolean,
  isAdminModifiable: boolean,
  commentAdded: string
  accountPreview: AccountPreviewDto,
}
